react-native-community/react-native-masked-view
# feat(#162): fabric support [reintroduce]

# Overview




The [PR](https://github.com/react-native-masked-view/masked-view/pull/221) with fabric support for RN >= 0.71 was reverted a few days ago, because of lint issues on pipeline. This PR reintroduces fabric changes with lint issues fixed
cc: @brentvatne 

# Test Plan



1. Install dependencies with `yarn` in the root directory
2. Run lint, flow and typescript to check code quality
- `yarn test:lint`
- Run `yarn test:flow`
- Run `yarn test:typescript`
3. Run `yarn` in the `example` directory
4. Test the lib on Android old arch
- Set the `newArchEnabled=false` in `example/android/gradle.properties`
- Run the app on Android
5. Test the lib on Android new arch
- Run the `./gradlew clean` in `example/android`
- Set the `newArchEnabled=true` in `example/android/gradle.properties`
- Run the app on Android
6. Test the lib on iOS old arch
- Install Pods (`npx pod-install`)
- Run the app on iOS
7. Test the lib on iOS new arch
- Clean the build in XCode
- Install Pods with `RCT_NEW_ARCH_ENABLED` flag (`RCT_NEW_ARCH_ENABLED=1 npx pod-install`)
- Run the app on iOS
